pylint/checkers/deprecated.py

```diff
@@ -214,9 +214,13 @@ def deprecated_attribute(self) -> Container[str]:
 
     def check_deprecated_attribute(self, node: astroid.Attribute) -> None:
         """Checks if the attribute is deprecated."""
-        qnames = {node.expr.name}
-        if any(name in self.deprecated_attribute() for name in qnames):
-            self.add_message("deprecated-attribute", node=node, args=(node.attrname,))
```

Comment:
This looks like it called deprecated_attributes() multiple times, but there's only one qname, so the any isn't needed, fixed below.